controllers/util/k8sutil.go

```diff
@@ -44,6 +45,10 @@ const (
 	OwnerUIDIndexName = "ownerUID"
 )
 
+func GetCrds() []string {
+	return []string{}
```

Comment:
is my understanding correct that in upcoming PRs we register the CRDNames in this and conditionally watch on them in the setupwithmanager?

Reply:
Added it with https://github.com/red-hat-storage/ocs-operator/pull/2745